openfecwebapp/templates/legal-advisory-opinion-landing.html

```diff
@@ -0,0 +1,53 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/legal.html' as legal %}
+{% import 'macros/breadcrumbs.html' as breadcrumb %}
+
+{% set breadcrumb_links=[(cms_url + '/legal-resources', 'Legal resources')] %}
+
+{% block title %}Search advisory opinions{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  {{ breadcrumb.breadcrumbs('Advisory opinions', breadcrumb_links) }}
+</header>
+
+<section class="main__content--full data-container__wrapper">
+  {{ legal.filter('Search advisory opinions', 'advisory_opinions') }}
+  <div class="content__section data-container__body">
```

Comment:
And on mobile there's no padding: 
![image](https://cloud.githubusercontent.com/assets/1696495/17614068/0485e080-6018-11e6-8e59-ca7702ed7085.png)